### ISSUE TITLE
Cannot retrieve song info

### ISSUE THREAD

**Issue by Putnam3145** [Tornith/CHSplit#1]
**Describe the bug**
CHsplit simply shows a blank screen instead of splits when a song is started, regardless of the song. The logs have this info

```
2020-02-09 11:44:00,444  INFO     DEBUG MODE: true
2020-02-09 11:44:00,447  INFO     Main loop started
2020-02-09 11:44:00,447  INFO     State: preinit, call: on_entry
2020-02-09 11:44:00,487  INFO     Manual data request: state
2020-02-09 11:44:00,489  DEBUG    state: preinit
2020-02-09 11:44:00,489  DEBUG    Sent REQUEST_RESPONSE_STATE
	Message: {"state": "preinit"}
2020-02-09 11:44:00,546  INFO     State: preinit, call: on_exit
2020-02-09 11:44:00,546  INFO     State: init, call: on_entry
2020-02-09 11:44:00,546  DEBUG    D:\Games\clonehero-win64\CHSplit/offsets/offsets.23_2_2.yml
2020-02-09 11:44:00,579  INFO     Manual data request: offset_list
2020-02-09 11:44:00,604  DEBUG    offset_list: [{'file_category': 'Clone Hero', 'game_label': 'v23.2.2', 'game_version': '23_2_2'}]
2020-02-09 11:44:00,604  DEBUG    Sent REQUEST_RESPONSE_OFFSET_LIST
	Message: {"offset_list": [{"file_category": "Clone Hero", "game_label": "v23.2.2", "game_version": "23_2_2"}]}
2020-02-09 11:44:01,187  INFO     Successfully attached to the game process. Base address: 7ff71f450000
2020-02-09 11:44:01,187  INFO     State: init, call: on_exit
2020-02-09 11:44:01,188  INFO     State: menu, call: on_entry
2020-02-09 11:44:08,167  INFO     State: menu, call: on_exit
2020-02-09 11:44:08,167  INFO     State: pregame, call: on_entry
2020-02-09 11:44:08,275  ERROR    Couldn't retrieve song info
2020-02-09 11:44:08,381  ERROR    Couldn't retrieve song info
... repeat the above every ~100 milliseconds until CHSplit is closed
```

**To Reproduce**
It seems to work for other people, so I wouldn't be able to give reproduction steps. I have restarted both CH and CHSplit.

**Expected behavior**
I expect the program to attach to the process (this happens correctly), detect when a song is playing (also happens correctly) and retrieve the song info from memory (not happening).

**Screenshots**
![image](https://user-images.githubusercontent.com/3927690/74108713-ebd05080-4b31-11ea-966a-79b0341212ba.png)

**Desktop (please complete the following information):**
 - OS: Window 10 64-bit
 - Browser: Firefox
 - Version: 0.2

**Additional context**
None, really, I've tried this with two installs of CH in two separate locations and both fail to work.

**Comment by Tornith** [comment]
What CH version are you running and do you have any mods?

**Comment by Putnam3145** [comment]
Updated to latest (0.23.2.2) in the process of troubleshooting, no mods, I tested this on a completely fresh install to make sure.

**Comment by TakaDoesStuff** [comment]
For me this happens randomly, everything worked first try (and when it works, it's good for the whole session) but it seems the more splits I have saved when I start CHSplit the more likely it is to break. I now have some saved for about 70 different songs and the chance of not getting this bug has sunk so low that I just gave up after 20 attempts

**Comment by TakaDoesStuff** [comment]
So I've found a solution, it'll sound like Placebo effect but I've tested often enough that I'm confident it's reliable.

Start CHSplit and immediately afterwards (before the window has even opened) start CH. Then do nothing other than plugging in your guitar until both of them have completely finished booting up.

(I then also need to alt+tab once or CHSplit doesn't stay on top for some reason but that's probably not part of the solution)
